sklearn/compose/_column_transformer.py

```diff
@@ -1320,6 +1320,21 @@ def get_metadata_routing(self):
 
         return router
 
+    def _more_tags(self):
+        return {
+            "_xfail_checks": {
```

Comment:
`ColumnTransformer` is now tested along all other estimators,hence more checks are failing and need to be ignored, and fixed later.